packages/prices/utils/price.ts

```diff
@@ -2,13 +2,11 @@
  * External dependencies
  */
 import { CURRENCY } from '@woocommerce/settings';
-import type { CurrencyResponseInfo } from '@woocommerce/type-defs/cart-response';
-import type { CartShippingPackageShippingRate } from '@woocommerce/type-defs/cart';
-
-/**
- * Internal dependencies
- */
-import type { Currency } from '../types';
+import type {
+	Currency,
+	CurrencyResponse,
+	CartShippingPackageShippingRate,
+} from '@woocommerce/types';
 
 type SymbolPosition = 'left' | 'left_space' | 'right' | 'right_space';
```

Comment:
Maybe we should move this to the `@woocommerce/types` package too and update https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/5261c4e9a733d5292ab40027a6ad6b3d61952e73/assets/js/settings/shared/settings-init.ts#L15 to use this type instead of the strings